.gitmodules

```diff
@@ -1,7 +1,6 @@
 [submodule "mavlink/include/mavlink/v2.0"]
 	path = mavlink/include/mavlink/v2.0
-	url = https://github.com/mavlink/c_library_v2.git
-	branch = master
```

Comment:
Do you think we should create a branch here?  I only worry if master gets active updates, if it's a slow moving repo than likely OK.